### ISSUE TITLE
Codegen doesn't use "example" for response body

### ISSUE THREAD

**Issue by pcads** [swagger-api/swagger-codegen#2387]
Not sure if this is by design or bug.
I have example for every object under definitions section. When I use codegen to generate a nodejs server, I am hoping at least the response body for GET should use the example data, but it doesn't. 

Do I miss anything? Thanks

**Comment by wing328** [comment]
@pcads I don't think the nodejs server stub generator supports the feature you mentioned and we welcome contribution to make the improvement. 

The msutache template for `nodejs` is a good starting point:

https://github.com/swagger-api/swagger-codegen/tree/master/modules/swagger-codegen/src/main/resources/nodejs

**Comment by pcads** [comment]
I am not familiar with mustache. I will take a look and hopefully I could do something. Thanks

**Comment by wing328** [comment]
My apologies. In the `nodejs` template, I did find the use of {{examples}} mustache tag:
```
service.mustache:  {{#examples}}examples['{{contentType}}'] = {{{example}}};
service.mustache:  {{/examples}}
```
Can you try starting the nodejs server with the following command instead?
```
NODE_ENV=development node index
```

**Comment by pcads** [comment]
I tried with NODE_ENV=development node index, still doesn't work.

**Comment by djskinner** [comment]
Yeah I don't think using the example data is in the node template. I was using it earlier and the controllers serve up dummy data that contains the likes of  `{"userName": "aeiou"}`
